### DISCUSSION TITLE
writing permission error

### DISCUSSION THREAD

**Discussion by replicare** [sourcebroker/deployer-extended-typo3#23] [General]
Dear Deployer-Team,

I am using your tool on an exsiting Project. (Former TER Typo3 10.4) I have configured everything and succsessfully deployed to my staging/live server. I have imported the old DB and all relevant files. - It works so far, but I have issues with writing errors in frontend like:

(1/1) #1390290029 TYPO3\CMS\Core\Resource\Exception\InsufficientFolderReadPermissionsException
You are trying to use a method on the inaccessible folder "_processed_".

Typo3 Directory Status:
Directory / is not writable
Path /var/www/releases/35 exists, but no file underneath it can be created.

I have tried to change the wrriteble mode in deployer.php

#set('writable_recursive',false);
set('writable_chmod_mode','0775');
#set('writable_dirs', ['public/typo3conf', 'public/typo3temp', 'public/uploads']);
set('writable_dirs', ['var', 'public/typo3conf', 'public/typo3temp']);

but the error is still exsiting. What I am doing wrong? 

Thank you very much in advance for your help. Any hint would help. It is quite urgent ..

**Comment by kszymukowicz** [comment]
hi

Its more general question for deployer than to deployer-extended-typo3. So maybe better ask on deployer github project directly.

I assume that the ssh user is different from apache user. Just compare owner/group for files created with ssh and those files created from apache level (try to see files created in typo3temp).
